fieldsolver/ldz_main.cpp

```diff
@@ -396,5 +397,9 @@ bool propagateFields(
    
    calculateVolumeAveragedFields(perBGrid,EGrid,dPerBGrid,volGrid,technicalGrid);
    calculateBVOLDerivativesSimple(volGrid, technicalGrid, sysBoundaries);
+   if(FieldTracing::fieldTracingParameters.doTraceFluxRopes || Parameters::computeCurvature) {
+      volGrid.updateGhostCells();
```

Comment:
calculatBVOLDerivativesSimple does one *before* the derivatives are computed, I need the up-to-date derivatives here. And the curvature is stored on that same volgrid so we need to update it again to have the curvature up to date for the gridGlue.

SHould this be split? Are we approaching the edge of the abyss and finally have to make some container of fsgrids instead of passing around many different smaller ones?